tools/windows-node-installer/README.md

```diff
@@ -41,10 +41,13 @@ Available Commands:
 
 ```bash
 ./wni aws create --kubeconfig <path to OpenShift cluster>/kubeconfig --credentials <path to aws>/credentials 
---credential-account default --image-id ami-06a4e829b8bbad61e --instance-type m4.large --ssh-key <name of the 
+--credential-account default --image-id ami-06a4e829b8bbad61e --instance-type m5a.large --ssh-key <name of the 
```

Comment:
just a final question, should we also update the instance-type in the aws e2e test file.? https://github.com/openshift/windows-machine-config-operator/blob/master/tools/windows-node-installer/test/e2e/aws_test.go#L28

Reply:
The e2e test for WNI does not involve adding it to a cluster so it is fine to leave it as is.